lib/index.ts

```diff
@@ -170,7 +170,11 @@ export default class PointerTracker {
     // Add listeners for additional events.
     // The listeners may already exist, but no harm in adding them again.
     if (isPointerEvent(event)) {
-      this._element.setPointerCapture(event.pointerId);
+      const capturingElement = event.target ?? this._element;
+      if (!(capturingElement instanceof Element)) {
+        throw new Error('Expected the event target to be an Element.');
+      }
```

Comment:
How about?

```suggestion
      const capturingElement =
        event.target && 'setPointerCapture' in event.target
          ? event.target
          : this._element;
```

Avoids the need to throw an error.

Reply:
Great idea